src/css/grapesjs.scss

```diff
@@ -90,6 +90,54 @@
   right: var(--viewsPanelWidth);
 }
 
+// Sticky CSS tags section
+
+.gjs-pn-views-container > div > div:first-child > div:first-child {
```

Comment:
This is not a good way to select the element you want
Use grapesjs API to get the element you want (or at least one of its close containers) and add a css class to it, e.g get the container with `silex.getEditor()
.Panels.getPanel('views-container')`
Please look for a better way

Reply:
Yeah, I knew it wasn't ideal, thank you for the guidance. Will do this.

Reply:
https://github.com/silexlabs/Silex/pull/1580/commits/07dd4e03e735d5d4aa63a27fc05fb08753b91ea2
Please review this (thank you for your patience, despite my technical issues today!)